packages/core/src/components/autosuggest/autosuggest.tsx

```diff
@@ -331,39 +343,44 @@ export class Autosuggest {
     return (
       <>
         <slot />
-        <ul
-          role="listbox"
-          id={this.listboxId}
-          aria-labelledby={this.labelId}
-          ref={element => this.listbox = element}
-          hidden={!this.showSuggestions}
-        >
-          {this.showSuggestions
-            ? this.suggestions.map((suggestion) => (
-                <li
-                  role="option"
-                  id={this.listboxItemId(suggestion)}
-                  key={suggestion.value}
-                  onMouseEnter={() => this.selectSuggestion(suggestion)}
-                  onMouseLeave={() => this.resetSelectedSuggestion()}
-                  onClick={() => this.pickSelectedValue()}
-                  aria-selected={(suggestion === this.selectedSuggestion).toString()}
-                  aria-label={suggestion.value}
-                >
-                  <span class="value">
-                    {this.markTerms(suggestion.value, terms)}
-                  </span>
-                  {suggestion.type
-                    ? (
-                      <span class="type">{suggestion.type}</span>
-                    )
-                    : undefined
-                  }
-                </li>
-              ))
-            : undefined
-          }
-        </ul>
+        {this.loading
+          ? <div class="autosuggest-progress-box">
+              <dso-progress-indicator label={this.loadingLabel}></dso-progress-indicator>
+            </div>
+          : <ul
+              role="listbox"
+              id={this.listboxId}
+              aria-labelledby={this.labelId}
+              ref={element => this.listbox = element}
+              hidden={!this.showSuggestions || this.loading}
```

Comment:
Volgens mij is deze `|| this.loading` niet nodig, in deze branch kan `this.loading` nooit meer true zijn.